README.md

```diff
@@ -1,2 +1,262 @@
-# hello-rails-back-end
-"Hello World" sample project with two apps. This is the API made in Rails.
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://github.com/Tarikul-Islam-Anik/Animated-Fluent-Emojis/blob/4eb68a32315bb30adfc08f423052bbf09b955cd4/Emojis/Hand%20gestures/Waving%20Hand%20Light%20Skin%20Tone.png" alt="wave" width="120" height="auto" />
+  <br/>
+
+### About
+**Hello Rails React** is a sample project that shows how to integrate Rails and React. [See more...](#about-project)
+
+</div>
+
+<div align="center"><hr width="250px"/></div>
+
+# 📗 Table of Contents
+
+- [💬 About the project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Usage](#usage)
+  - [Run](#run)
+  - [Open](#open)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<div align="center"><hr width="250px"/></div>
+
+# 👋**Hello Rails React** <a name="about-project"></a>
+
+  **Hello Rails React** is a sample project that shows how to integrate Rails and React.
+  It is composed of two projects, one for the back end and another one to for the front end.
+  
+  ### Ruby on Rails
+  - RoR is used for the back end, and it is configured as an api, so it attends the requests of the front end.
+  - [`hello-rails-back-end`](https://github.com/luigirazum/hello-rails-back-end) this repository, is the one that holds the RoR project.
+
+  ### React
+  - React is used for the front end, and it consumes the back end as an API.
+  - [`hello-rails-front-end`](https://github.com/luigirazum/hello-rails-front-end), is the repository that holds the React project.
```

Comment:
- [ ]  Kindly fix this broken link to in order to make your README file more professional. I got 404 error. 👍

Reply:
Thanks for you feedback @OyePriscilla.
The required changes were applied within this [commit](https://github.com/luigirazum/hello-rails-back-end/pull/1/commits/7fe16cc8a394125c09213a2eaf4100ebae921003)